backend/hitas/views/apartment_max_price.py

```diff
@@ -123,6 +126,36 @@ def update(self, request, *args, **kwargs):
             ampc.delete()
             return Response(status=HTTPStatus.NO_CONTENT)
 
+    @action(detail=True, methods=["GET"])
+    def download(self, request, **kwargs) -> HttpResponse:
+        with model_or_404(ApartmentMaximumPriceCalculation):
+            mpc = ApartmentMaximumPriceCalculation.objects.get(uuid=kwargs["pk"])
+
+            # Migrated maximum price calculations are missing necessary data necessary to create a PDF
+            if mpc.json_version is None:
+                raise HitasModelNotFound(model=ApartmentMaximumPriceCalculation)
+
+            # Make sure the calculation is confirmed
+            if mpc.confirmed_at is None:
+                return Response(
+                    {
+                        "error": "not_confirmed",
```

Comment:
this error should be added to OpenAPI but as this is getting removed(?) it does not matter that much :thinking:

Reply:
This is getting removed? 🤔 
Or replaced with another error later on?